src/persist-client/src/cfg.rs

```diff
@@ -398,6 +399,13 @@ pub const CRDB_TCP_USER_TIMEOUT: Config<Duration> = Config::new(
     connection is forcibly closed.",
 );
 
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const TXN_USE_CRITICAL_SINCE: Config<bool> = Config::new(
```

Comment:
fyi, persist-txn does have [its own set of dyncfgs](https://github.com/MaterializeInc/materialize/blob/c6763906f22f3924ea2b33f3aec7a40b03f1fe73/src/persist-txn/src/lib.rs#L265-L272). not worth a followup PR to fix though